pytest.ini

```diff
@@ -3,7 +3,6 @@ addopts = -l
 junit_family=xunit2
 
 filterwarnings =
-    error
     always::scipy._lib._testutils.FPUModeChangeWarning
```

Comment:
```suggestion
    error
    always::scipy._lib._testutils.FPUModeChangeWarning
```

Reply:
I think I agree with you and others that this is a bit wild, even this doesn't help for me locally:

```diff
--- a/pytest.ini
+++ b/pytest.ini
@@ -3,14 +3,4 @@ addopts = -l
 junit_family=xunit2
 
 filterwarnings =
-    error
-    always::scipy._lib._testutils.FPUModeChangeWarning
-    ignore:.*deprecated and ignored since IPython.*:DeprecationWarning
-    once:.*LAPACK bug 0038.*:RuntimeWarning
-    ignore:can't resolve package from __spec__ or __package__, falling back on __name__ and __path__:ImportWarning
-    ignore:assertions not in test modules or plugins:pytest.PytestConfigWarning
-    ignore:'environmentfilter' is renamed to 'pass_environment'
-    ignore:'contextfunction' is renamed to 'pass_context'
-    ignore:.*The distutils.* is deprecated.*:DeprecationWarning
-    ignore:\s*.*numpy.distutils.*:DeprecationWarning
-    ignore:.*The --rsyncdir command line argument.*:DeprecationWarning
+    error::DeprecationWarning
```

Sounds like Stefan figured it out though--likely related to NumPy C-level control of warnings rather than higher-level warns.